client/rpc/Cargo.toml

```diff
@@ -37,6 +37,7 @@ sp-transaction-pool = { version = "2.0.0-rc5", path = "../../primitives/transact
 sp-blockchain = { version = "2.0.0-rc5", path = "../../primitives/blockchain" }
 hash-db = { version = "0.15.2", default-features = false }
 parking_lot = "0.10.0"
+lazy_static = "1.4.0"
```

Comment:
It's probably not worth the extra complexity but this should have been declared as an optional dependency (still kept in `dev-dependencies`), and enabled on `test-helpers`.

Reply:
Is it possible to mark `dev-dependencies` as `optional`? I at least made it `optional` (while still in `dependencies`)